src/__tests__/companies.test.js

```diff
@@ -63,7 +63,7 @@ describe('companiesSlice tests', () => {
     fetchApi.mockResolvedValue(mockData);
     await store.dispatch(fetchCompaniesThunk('symbol'));
 
-    render(
+    const { yearInfo } = render(
       <Provider store={store}>
         <MemoryRouter initialEntries={['/AAPL/2022']}>
           <Routes>
```

Comment:
- [ ] As requested by the previous reviewer, Please add tests for all components in the `/components` directory.